Library/ViewModels/ManagePledgeViewModelTests.swift

```diff
@@ -114,27 +110,80 @@ internal final class ManagePledgeViewModelTests: TestCase {
   func testConfigurePaymentMethodViewController() {
     self.configurePaymentMethodView.assertDidNotEmitValue()
 
-    let backing = Backing.template
-
     let project = Project.template
-      |> \.personalization.backing .~ backing
+      |> Project.lens.personalization.backing .~ Backing.template
 
-    self.vm.inputs.configureWith(project)
+    let envelope = ManagePledgeViewBackingEnvelope.template
 
-    self.vm.inputs.viewDidLoad()
+    let mockService = MockService(
+      fetchManagePledgeViewBackingResult: .success(envelope)
+    )
+
+    let pledgePaymentMethodViewData: ManagePledgePaymentMethodViewData = (
+      backingState: .pledged,
+      expirationDate: "2020-01-01",
+      lastFour: "1234",
+      creditCardType: .visa,
+      paymentType: .creditCard
+    )
 
-    self.configurePaymentMethodView.assertValue(Backing.template)
+    withEnvironment(apiService: mockService) {
+      self.vm.inputs.configureWith(project)
+
+      self.vm.inputs.viewDidLoad()
+
+      self.configurePaymentMethodView.assertDidNotEmitValue()
+
+      self.scheduler.advance()
+
+      self.configurePaymentMethodView.assertValueCount(1)
+
+      XCTAssertTrue(
+        self.configurePaymentMethodView.values.map { $0 == pledgePaymentMethodViewData }.allSatisfy(isTrue)
```

Comment:
Yikes, maybe this is a good reason to use `struct`s instead of tuples for these view-specific "models".